cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -175,6 +175,24 @@ spec:
         - mountPath: /run/xtables.lock
           name: xtables-lock
           readOnly: false
+      initContainers:
+      - name: init-kube-proxy
+        image: gcr.io/google_containers/busybox:1.27.2
+        command:
+        - /bin/sh
+        - -c
+        - 'echo $(PROXY_CONFIG) | grep "mode: ipvs" && modprobe -a
+          ip_vs ip_vs_rr ip_vs_wrr ip_vs_sh nf_conntrack_ipv4 || echo "did not load
```

Comment:
More comments, do we really need this init container? As the kube-proxy already modprobe the required modules at here https://github.com/kubernetes/kubernetes/blob/release-1.9/pkg/proxy/ipvs/proxier.go#L697-L703 for 1.9?

Reply:
As we discussed in  #52003,  Folks believed a better solution is to load kernel modules in init container instead as that has less security concerns, but use this as an alternative solution in first to unblock the IPVS beta work in rebase-1.9.

Reply:
Thanks @Lion-Wei 

1) Can you please clarify more for the security risk? and how does initContainer can help this?
2) Why `modprobe` is needed in initContainer? I see that the kube-proxy can load those modules dynamically.